src/ethernet_interface.cpp

```diff
@@ -682,6 +683,58 @@ void EthernetInterface::loadNameServers(const config::Parser& config)
         config.map.getValueStrings("Network", "DNS"));
 }
 
+void EthernetInterface::loadStaticRoutes(const config::Parser& config)
+{
+    std::vector<std::string> destinations =
+        config.map.getValueStrings("Route", "Destination");
+    std::vector<std::string> gateways =
+        config.map.getValueStrings("Route", "Gateway");
+    for (uint8_t i = 0; i < destinations.size() && i < gateways.size(); i++)
+    {
+        size_t pos = destinations[i].find("/");
+        std::string dest = destinations[i].substr(0, pos);
```

Comment:
Should we validate the IP and the prefix here?

Reply:
validation will done here
```
  try
        {
            ifaddr = {addr, prefix};
        }
        catch (const std::exception& e)
        {
            auto msg = fmt::format("Invalid static route {}\n", e.what());
            log<level::ERR>(msg.c_str());
        }
```

Reply:
Okay